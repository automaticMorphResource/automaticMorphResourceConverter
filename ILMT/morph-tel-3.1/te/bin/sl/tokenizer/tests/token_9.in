బ్రిస్బేన్ నగరానికి దగ్గరగా ఉండే కెయిర్న్స్ అనే చిన్న ఊరు నుంచి జలాంతర్గామిలో సముద్రంలోకి కొంత దూరం ప్రయాణిస్తే ఈ గ్రేట్ బారియర్ రీఫ్ ని చేరుకోవచ్చు. 
