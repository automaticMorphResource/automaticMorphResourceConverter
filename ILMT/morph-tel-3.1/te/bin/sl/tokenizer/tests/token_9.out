<Sentence id="1">
1	బ్రిస్బేన్	unk
2	నగరానికి	unk
3	దగ్గరగా	unk
4	ఉండే	unk
5	కెయిర్న్స్	unk
6	అనే	unk
7	చిన్న	unk
8	ఊరు	unk
9	నుంచి	unk
10	జలాంతర్గామిలో	unk
11	సముద్రంలోకి	unk
12	కొంత	unk
13	దూరం	unk
14	ప్రయాణిస్తే	unk
15	ఈ	unk
16	గ్రేట్	unk
17	బారియర్	unk
18	రీఫ్	unk
19	ని	unk
20	చేరుకోవచ్చు	unk
21	.	unk
</Sentence>

